lib/bike.dart

```diff
@@ -30,7 +30,7 @@ class Bike extends _$Bike {
       _updateTimer?.cancel();
       _updateDebounce?.cancel();
     });
-    _updateTimer = Timer.periodic(const Duration(seconds: 10), (timer) {
+    _updateTimer = Timer.periodic(const Duration(seconds: 5), (timer) {
```

Comment:
Sigh, would be real nice to get subscriptions working. One day...

Reply:
Maybe next update patch, as there is a lot of tinkering with it.

Will probably play with it later when GF needs approves the bike VISA in the apartment again,
*inside joke.